### ISSUE TITLE
Bug: Selecting an Overdue task does not bring up the note. Selecting non-overdue tasks does.

### ISSUE THREAD

**Issue by dokmanc** [uphy/obsidian-reminder#105]
### Describe the bug

In the Reminders window, my tasks are broken down in three groups, 1) Overdue (2) Today (3) By specific future date groupings.  if i click on any task item in any one of these groupings other than "Overdue", the note which contains the task is brought up. However, if I click on a task in the Overdue group, nothing happens, so I have to search like to fool by the task text to try to find the note to complete it, because I can't complete the task from the reminder window and I can't copy the text either to assist in searching for it. This shortcoming very inconvenient.

### Expected Behavior

I would expect that the note which contains the overdue task would open.

### Steps to reproduce

1. Create an overdue task. 
2. Open the Reminder window if not already open
3. Click on the task. 
4. Observe t hat nothing happens  as a result of step 3.

### Operating system

Windows

**Comment by ellannjohnson** [comment]
Experiencing the same/similar issue on Windows 11 & MacOS 12.3 

Windows
1. Selecting an overdue reminder does not bring up the reminder. 
2. When a linked note is included in a reminder, selecting the linked note brings a dialogue box asking to select a program to open the file type.  In this case, the file type is md. 

Mac
1. Selecting an overdue reminder does not bring up the reminder. 
2. Selecting a linked note included in a reminder brings a dialogue box "Link: app://obsidian.md/<page name>" Selecting a tag will bring a similar dialogue box "Are you sure you want to open this link? Link: a://obsidian.md.index.xml#<tag>"

I have not updated my android or iOS devices and I use a separate .obsidian folder for mobile. If I update, I will try to reproduce the issue and update here.

**Comment by mjs271** [comment]
FWIW, the behavior described by @dokmanc and `1.` in @ellannjohnson's, above is described in the [docs](https://uphy.github.io/obsidian-reminder/guide/list-reminders.html#overdue-reminders), making it seem like intended behavior.

Personally, I would still consider it unwanted behavior, as I can't think of a situation when I want to manually trigger a notification and would always rather be taken to the note so I can see the description or mark as completed.

**Comment by mjs271** [comment]
On further investigation, it looks like it may be a side-effect of using system notifications, as described in #5. However, I'm going to do some tinkering and will submit a PR if I can get it working.

**Comment by Harminder-Singh-Nijjar** [comment]
It would be great to have the ability to click on over-due reminders so that we can easily find and complete the task. Right now, we have to search for the task by its text, which can be inconvenient.

**Comment by adis-io** [comment]
I currently have system notifications enabled, so when I click on overdue, a system notifications opens, and when I click on that, an app popup appears.

**Comment by stoweboyd** [comment]
> On further investigation, it looks like it may be a side-effect of using system notifications, as described in #5. However, I'm going to do some tinkering and will submit a PR if I can get it working.

Yes,  I turned off system notifications and overdue tasks when clicked now open the source notes. So I have to accept no system notifications until this is fixed.

**Comment by foss-guru** [comment]
I experienced this bug on Android when the overdue note triggers an in-app popup notification. When I tap the note link displayed in the popup, it just disappears and brings me to the homescreen or last displayed note. This is a bit of a task automation show stopper when trying to get to the note.

Would also be worth noting, the popup notification for an overdue task suddenly stopped working for me on mobile (Android) but it was working when I first installed. Idk if there is an issue listed for that already. (If someone could verify that would be good) All I seemed to find was somebody asking about disabling popups which would be sort of the inverse of my issue.
